x-pack/test/functional_basic/apps/ml/data_visualizer/index_data_visualizer_actions_panel.ts

```diff
@@ -4,7 +4,7 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
-import { FtrProviderContext } from '../../../ftr_provider_context';
+import { FtrProviderContext } from '../../../../functional/ftr_provider_context';
```

Comment:
The `functional_basic` tests bring their own `ftr_provider_context` (which is currently imported from `functional`, but could be different in the future), so no need to import directly from `functional`.
This should work:
```
import { FtrProviderContext } from '../../../ftr_provider_context';
```